app/src/main/java/org/fossasia/openevent/general/order/OrdersUnderUserViewModel.kt

```diff
@@ -20,8 +20,8 @@ class OrdersUnderUserViewModel(
 
     private val compositeDisposable = CompositeDisposable()
     private lateinit var order: List<Order>
-    private val mutableAttendeesNumber = MutableLiveData<ArrayList<Int>>()
-    val attendeesNumber: LiveData<ArrayList<Int>> = mutableAttendeesNumber
+    private val mutableAttendeesNumber = MutableLiveData<List<Int?>>()
+    val attendeesNumber: LiveData<List<Int?>> = mutableAttendeesNumber
```

Comment:
Remove nullable Int

Reply:
No it won't require casting. Just because the IDE tells you it's the quick fix doesn't mean it's the correct fix. Use Elvis operator to use default value in case of null